### ISSUE TITLE
Error: Cannot find module './build/Release/shell'

### ISSUE THREAD

**Issue by bhardin** [percyhanna/mincer-ruby-sass#2]
Given `package.json`:
```json
{
  "name": "test",
  "version": "1.0.0",
  "devDependencies": {
    "mincer": "~1.2.2",
    "mincer-jsx": "~0.0.5",
    "mincer-ruby-sass": "~1.0.1"
  }
}
```

```bash
bhardin@brett-mbp ~/tmp  ᐅ node
> require('mincer-ruby-sass');
Error: Cannot find module './build/Release/shell'
    at Function.Module._resolveFilename (module.js:336:15)
    at Function.Module._load (module.js:278:25)
    at Module.require (module.js:365:17)
    at require (module.js:384:17)
    at Object.<anonymous> (/Users/bhardin/tmp/node_modules/mincer-ruby-sass/node_modules/execSync/index.js:30:11)
    at Module._compile (module.js:460:26)
    at Object.Module._extensions..js (module.js:478:10)
    at Module.load (module.js:355:32)
    at Function.Module._load (module.js:310:12)
    at Module.require (module.js:365:17)
```

**Comment by bhardin** [comment]
I'm thinking it's related to https://github.com/mgutz/execSync/issues/27.

When I attempt `npm install` with `mincer-ruby-sass`, I receive the following:

```bash
bhardin@brett-mbp ~/Code/mincer-ruby-sass (master ✔) ᐅ npm install

> execSync@1.0.2 install /Users/bhardin/Code/mincer-ruby-sass/node_modules/execSync
> node install.js

[execsync v1.0.2] Attempting to compile native extensions.
[execSync v1.0.2]
    Native code compile failed!!
npm WARN excluding symbolic link test/fixtures/app/views/.#application.js.coffee.erb -> sam@pursuit.41266
temp@0.8.1 node_modules/temp
└── rimraf@2.2.8

mincer@1.0.5 node_modules/mincer
├── shellwords@0.1.0
├── mimoza@0.3.0
├── hike@0.1.4
├── lodash@2.4.1
├── fs-tools@0.2.11 (async@0.2.10)
├── pako@0.2.5
├── argparse@0.1.16 (underscore@1.7.0, underscore.string@2.4.0)
└── source-map@0.1.43 (amdefine@0.1.0)

execSync@1.0.2 node_modules/execSync
└── temp@0.5.1 (rimraf@2.1.4)
```

**Comment by bhardin** [comment]
So. This has to do with `execsync` as predicted. However, I don't think we need to use `execsync` anymore because asynchronous I/O has been included in Node v0.12. So rather than require a library which is no longer maintained, we can use the core functionality.

**Comment by percyhanna** [comment]
Fixed in #4 by removing dependency on `execSync` and any native extensions.

**Comment by sowminiva** [comment]
use "grunt githooks" to resolve the issue.
